### PR TITLE
New/facet ordering

### PR DESCRIPTION
**Pull request by matthew1809**
Please check if the PR fulfills these requirements
 The commit message follows our guidelines YES
 Tests for the changes have been added (for bug fixes / features) NA
 Docs have been added / updated (for bug fixes / features) NO
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...) 
- Dynamic facet ordering

What is the current behavior? (You can also link to an open issue here)
Facets were ordered the same for every page

What is the new behavior (if this is a feature change)?
On the jeans page, the facets change according to a rule in the visual editor

Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
NO

Other information:

### CODE REVIEW

File: html/categoryPageAccessories.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            </div>
+            <div id="smart-sort-banner" class="smart-sort-banner"></div>
+            <div class="container-result-wrapper">
+                <p class="hideFilters">HIDE</p>
+                <div class="left-panel">
+                    <div id="clear-refinements" class="clear-refinements"></div>
+                    <div id="current-refinements" class="current-refinements"></div>
+                    <div id="dynamic-widgets"></div>
+            
+                    <!-- <div class="gender-wrapper filter-item-wrapper">
```
Comment (Haroenv) [inline html/categoryPageAccessories.html]: what's the commented code for?

File: html/categoryPageJeans.html [hunk 2]
```diff
@@ -91,69 +87,27 @@
...
-                        </ul>
-                    </div> -->
-                    <div class="gender-wrapper filter-item-wrapper">
-                        <!-- <div class="btn-backFilter">
-                            <div><i class="fas fa-chevron-left"></i>Filters</div>
-                        </div> -->
+                    <div id="dynamic-widgets"></div>
+                    <!-- <div class="gender-wrapper filter-item-wrapper">
                         <h2>Gender</h2>
                         <div id="gender-list" class="gender-list"></div>
```
Comment (Haroenv) [inline html/categoryPageJeans.html]: there's more commented code here

File: html/index.html [hunk 3]
```diff
@@ -75,7 +75,7 @@
             <li><a href="./categoryPageWoman.html">WOMEN</a></li>
             <li><a href="./categoryPageMan.html">MEN</a></li>
             <li class="jeanbtn"><a href="./categoryPageJeans.html">JEANS</a></li>
-            <li><a href="">MAGAZINE</a></li>
+            <li><a href="./categoryPageAccessories.html">ACCESSORIES</a></li>
```
Comment (Haroenv) [inline html/index.html:78]: should it not be called just `accessories`

File: src/SearchResultPage/searchResults.js [hunk 4]
```diff
@@ -949,62 +468,68 @@ export function searchResults() {
...
+    }),
+    customCurrentRefinements({
+      container: document.querySelector('#current-refinements'),
+    }),
     customConfigure({
       container: document.querySelector('#configure'),
       searchParameters: {
         hitsPerPage: 20,
         enablePersonalization: true,
+        facetFilters: extraSearchFilters,
```
Comment (Haroenv) [inline src/SearchResultPage/searchResults.js]: maybe use `filters` so it doesn't overlap with facets set by the widgets?

File: src/SearchResultPage/searchResults.js [hunk 5]
```diff
@@ -949,62 +468,68 @@ export function searchResults() {
...
+
   search.addWidgets([
+    customQueryRuleCustomData({
+      container: document.querySelector('#banner'),
+    }),
+    customCurrentRefinements({
+      container: document.querySelector('#current-refinements'),
+    }),
     customConfigure({
       container: document.querySelector('#configure'),
```
Comment (Haroenv) [inline src/SearchResultPage/searchResults.js:485]: unrelated, but why is a configure displayed?

**Review by Haroenv — COMMENTED** [review]
dynamic widgets looks implemented correctly

File: html/index.html (hunk 3 above)
Comment (matthew1809) [reply html/index.html:78]: Screenshot 2021-07-27 at 10 44 40
 
Upper case makes sense as that is how they are all shown in the rendered HTML

File: src/SearchResultPage/searchResults.js (hunk 4 above)
Comment (matthew1809) [reply src/SearchResultPage/searchResults.js]: Done

File: html/categoryPageJeans.html (hunk 2 above)
Comment (matthew1809) [reply html/categoryPageJeans.html]: Resolved

File: html/categoryPageAccessories.html (hunk 1 above)
Comment (matthew1809) [reply html/categoryPageAccessories.html]: None, redundant, removed, good spot

File: src/SearchResultPage/searchResults.js (hunk 5 above)
Comment (matthew1809) [reply src/SearchResultPage/searchResults.js:485]: Maybe @HugoWith knows

File: html/index.html (hunk 3 above)
Comment (Haroenv) [reply html/index.html:78]: I meant the url

**Review by Haroenv — APPROVED** [review]
looks all right, the dynamic widgets is implemented correctly

**Pull request merged**